libshaderc/include/shaderc.h

```diff
@@ -236,7 +236,7 @@ typedef struct shaderc_spv_module* shaderc_spv_module_t;
 shaderc_spv_module_t shaderc_compile_into_spv(
     const shaderc_compiler_t compiler, const char* source_text,
     size_t source_text_size, shaderc_shader_kind shader_kind,
-    const char* entry_point_name,
+    const char* input_file_name, const char* entry_point_name,
```

Comment:
The blurb should change to describe the argument.  For one thing it's a null-terminated string.